docs/code/mecsimcalc-library.md

```diff
@@ -0,0 +1,155 @@
+---
```

Comment:
This doc page would replace this old one: https://github.com/MecSimCalc/MecSimCalc-docs/blob/main/docs/utility.md.
Can you delete the old one and make sure that all relevant info is copied over.

Reply:
There's also other places in the docs, where the old functions are used, like `plt_show` is defined and used here: https://github.com/MecSimCalc/MecSimCalc-docs/blob/main/docs/maps/example-2.md?plain=1#L201-L212.
Use ctrl+shift+f to do a search for them and replace instead by an import of your functions.

Reply:
Once you're done with the docs, we'd also have to update the tutorial code (when you create a new app) to use your library: https://github.com/MecSimCalc/MecSimCalc/tree/main/static/Templates.
To get the updated code, you would create a new app sing the template, make the changes in the code section and then download the app (json) from the apps table. Copy and past new fields from the json file